tests/integration/test_filter.py

```diff
@@ -157,12 +157,7 @@ def test_filter_year(c):
         (
             "SELECT * FROM parquet_ddf WHERE (b > 5 AND b < 10) OR a = 1",
             lambda x: x[((x["b"] > 5) & (x["b"] < 10)) | (x["a"] == 1)],
-            [
-                [("a", "==", 1), ("b", "<", 10)],
-                [("a", "==", 1), ("b", ">", 5)],
-                [("b", ">", 5), ("b", "<", 10)],
-                [("a", "==", 1)],
-            ],
+            [[("b", ">", 5), ("b", "<", 10)], [("a", "==", 1)]],
```

Comment:
Just checking; are these param changes related to disabling `FilterColumnsPostJoin`?

Reply:
I don't know why those changed? I don't think it was me that changed them unless I just don't remember. It should not be related to `FilterColumnsPostJoin` however.

Reply:
I will say the new change seems more correct than the original way anyway.

Reply:
I can comment a bit more on this. This isn't related to `FilterColumnsPostJoin`, but how datafusion optimizes predicates for table_scan operations. Starting df14, they started using cnf (conjunctive normal form) for the expressions being passed as filters. (See https://github.com/dask-contrib/dask-sql/pull/903#issuecomment-1302767610). 
Since this particular test already had filters in dnf, the conversion of dnf->cnf (by df) -> dnf (by dask-sql since that's what arrow expects) resulted in an overly complex filter.

df16/17 seems to simplify some of these expressions and doesn't convert everything to cnf.